src/tests/validateWithJSONSchema.test.ts

```diff
@@ -0,0 +1,39 @@
+import { Type } from '@sinclair/typebox'
+import { validateWithJSONSchema } from '../resolvers/validateWithJSONSchema'
+
+describe('validateWithJSONSchema', () => {
+  it('should validate input against a JSON schema', () => {
+    expect(
+      validateWithJSONSchema(
+        Type.Object({
+          foo: Type.String({ minLength: 1 }),
+        }),
+      )({ foo: 'bar' }),
+    ).toEqual({
+      value: { foo: 'bar' },
+      errors: undefined,
+    })
+  })
+  it('should return errors if input does not match the JSON schema', () => {
```

Comment:
nit: this will be easier for me to read with an empty line between the `it` calls. thank you!